homeassistant/components/alarm_control_panel/services.yaml

```diff
@@ -13,71 +13,46 @@ alarm_disarm:
 alarm_arm_custom_bypass:
   target:
     entity:
-      domain: alarm_control_panel
+      <<: *alarm_entity
```

Comment:
I'm not sure this is helpful or just annoying

Reply:
It's not much shorter so then it doesn't help.